src/inc/woocommerce-actions.php

```diff
@@ -120,7 +126,7 @@ public static function login_failure( string $username, \WP_Error $error ) {
 				$failure_reason = null;
 		}
 		$properties = array(
-			'$user_id'      => $attempted_user->ID ? $attempted_user->ID : null,
+			'$user_id'      => (string) $attempted_user->ID ?? null,
```

Comment:
Doesn't `$attempted_user->ID ?? null` on `(string)`?
I am always confused with that...